README.md

```diff
@@ -1,49 +1,177 @@
+| [CurseForge App][url-app-cf]                  |    [🔗&nbsp;Open][url-tutorial-cf]    |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |     ✅&nbsp;Yes      |
+| [Modrinth App][url-app-modrinth]              | [🔗&nbsp;Open][url-tutorial-modrinth] |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |     ✅&nbsp;Yes      |
+| [Prism Launcher][url-app-prism]               |  [🔗&nbsp;Open][url-tutorial-prism]   |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ✅&nbsp;Easy  |  ➖&nbsp;Partially   |
+| [MultiMC][url-app-multimc]                    | [🔗&nbsp;Open][url-tutorial-multimc]  |  ✅&nbsp;Easy  | ❌&nbsp;Manual | ❌&nbsp;Manual |  ➖&nbsp;Partially   |
+| [MultiMC auto-update][url-app-mmcu]           |   [🔗&nbsp;Open][url-tutorial-mmcu]   |  ✅&nbsp;Easy  | ❌&nbsp;Manual | ❌&nbsp;Manual |  ➖&nbsp;Partially   |
+| [Vanilla Minecraft Launcher][url-app-vanilla] | [🔗&nbsp;Open][url-tutorial-vanilla]  | ❌&nbsp;Manual | ❌&nbsp;Manual | ❌&nbsp;Manual |  ➖&nbsp;Partially   |
+| [GitHub Releases][url-app-github]             |            ➖&nbsp;_None_             | ❌&nbsp;Manual | ❌&nbsp;Manual | ❌&nbsp;Manual |      ❌&nbsp;No      |
+
+➖: These launchers support the modpack by increasing
+its download count, but not with monetary benefits.
+
+### MultiMC auto-update
+
+Download links for MultiMC auto-update:
+
+- [Minecraft 1.16.5](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.16.5.auto-update.zip)
+- [Minecraft 1.17.1](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.17.1.auto-update.zip)
+- [Minecraft 1.18.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.12.2/Fabulously.Optimized.MC.1.18.2.auto-update.zip)
+- [Minecraft 1.19.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v4.10.0/Fabulously.Optimized.MC.1.19.4.auto-update.zip)
+- [Minecraft 1.20.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.6.5/Fabulously.Optimized.MC.1.20.2.auto-update.zip)
+- [Minecraft 1.20.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.8.0-beta.11/Fabulously.Optimized.MC.1.20.4.auto-update.zip)
+
+### Resource Packs
+
+FO includes some resource packs [for a better experience][url-rp],
+and they can also be downloaded separately.
+
+| Resource Pack             | CurseForge                  | Modrinth                        | Planet Minecraft                   |
+| ------------------------- | --------------------------- | ------------------------------- | ---------------------------------- |
+| **Chat Reporting Helper** | [CurseForge][url-rp-crh-cf] | [Modrinth][url-rp-crh-modrinth] | [Planet Minecraft][url-rp-crh-pmc] |
+| **Fast Better Grass**     | [CurseForge][url-rp-fbg-cf] | [Modrinth][url-rp-fbg-modrinth] | [Planet Minecraft][url-rp-fbg-pmc] |
```

Comment:
As with MMC AU, I didn't really want to highlight this because FO users don't need it... but maybe it's fine.